docs/production/examples/fastapi.mdx

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 1
+title: FastAPI
+---
+
+# FastAPI example
+
+The example assumes you've already installed [**our SDK**](https://github.com/fishjam-cloud/python-server-sdk) and you're ready to go.  
+If you wish to see more general info, visit [**Setup your server**](/production/server) article.
```

Comment:
**issue (if-minor):** Out-of-scope, but I've just noticed that we're using a noun like a verb here.
```suggestion
If you wish to see more general info, visit [**Set up your server**](/production/server) article.
```